projects/cashmere-examples/src/project-template/src/app/cashmere.module.ts

```diff
@@ -0,0 +1,61 @@
+import {NgModule} from '@angular/core';
+import {
+    AccordionModule,
+    AppSwitcherModule,
+    BreadcrumbsModule,
+    BannerModule,
+    ButtonModule,
+    CheckboxModule,
+    ChipModule,
+    DrawerModule,
+    FormFieldModule,
+    IconModule,
+    InputModule,
+    ListModule,
+    ModalModule,
+    NavbarModule,
+    PaginationModule,
+    PicklistModule,
+    PopoverModule,
+    ProgressIndicatorsModule,
+    RadioButtonModule,
+    SelectModule,
+    SubnavModule,
+    TabsModule,
+    TableModule,
+    TileModule,
+    ToasterModule,
+    TypeformSurveyModule
+} from '@healthcatalyst/cashmere';
+
+@NgModule({
+    exports: [
+        AccordionModule,
+        AppSwitcherModule,
+        BreadcrumbsModule,
+        BannerModule,
+        ButtonModule,
+        CheckboxModule,
+        ChipModule,
+        DrawerModule,
+        FormFieldModule,
+        IconModule,
+        InputModule,
+        ListModule,
+        ModalModule,
+        NavbarModule,
+        PaginationModule,
+        PicklistModule,
+        PopoverModule,
+        ProgressIndicatorsModule,
+        RadioButtonModule,
+        SelectModule,
+        SubnavModule,
+        TableModule,
+        TabsModule,
+        TileModule,
+        ToasterModule,
+        TypeformSurveyModule
+    ]
+})
+export class CashmereModule {}
```

Comment:
as we already have a cashmere module in the root project, would it make sense to name this CashmereExamplesModule?